samples/scaleout/senderside/Version_6/AwareClient/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Messages;
+using NServiceBus;
+
+namespace AwareClient
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            BusConfiguration busConfig = new BusConfiguration();
```

Comment:
Can u rename to be consistent on busconfiguration. Will make it easier when we "endpointconfig"